main.py

```diff
@@ -0,0 +1,41 @@
+#first we define the essential: updater and dispatcher
+
+from telegram.ext import Updater
+updater = Updater(token  = '<INSERIRE TOKEN DEL BOT (vedi BotFather)>', use_context = True)
+dispatcher = updater.dispatcher
+
+import logging
+logging.basicConfig(format = '%(asctime)s - %(name)s - %(levelname)s - %(message)s', level = logging.INFO)
+
+#defining the introductory function
+from telegram import Update
+from telegram.ext import CallbackContext
+from telegram.ext import CommandHandler
+from telegram.ext import MessageHandler, Filters
+from telegram import ParseMode
+
+import random
```

Comment:
```suggestion
import logging
import random

from telegram import ParseMode
from telegram import Update
from telegram.ext import CallbackContext
from telegram.ext import CommandHandler
from telegram.ext import MessageHandler
from telegram.ext import Filters
from telegram.ext import Updater

updater = Updater(token  = '<INSERIRE TOKEN DEL BOT (vedi BotFather)>', use_context = True)
dispatcher = updater.dispatcher

logging.basicConfig(format = '%(asctime)s - %(name)s - %(levelname)s - %(message)s', level = logging.INFO)

#defining the introductory function

```
I think that not scattering the imports looks a bit better